<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Custom Notifications Demo</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="button_layout">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <widget class="QPushButton" name="register_button">
      <property name="text">
       <string>Register Notification</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="unregister_button">
      <property name="text">
       <string>Unregister Notification</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="force_unregister_button">
      <property name="text">
       <string>Forcibly Unregister Notifications</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="broadcast_button">
      <property name="text">
       <string>Broadcast Notification</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="channels">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>10</y>
     <width>290</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Click Broadcast Notification to send a callback.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="status_bar_layout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
